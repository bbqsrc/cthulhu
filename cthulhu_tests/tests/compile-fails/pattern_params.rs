#[cthulhu::invoke]
fn foo((_a, _b): (i32, bool)) {}

fn main() {}
