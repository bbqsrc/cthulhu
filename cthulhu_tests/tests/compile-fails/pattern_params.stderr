error: failed to invoke cthulhu: failed to convert Rust type to FFI type: Tuple parameters not supported
 --> $DIR/pattern_params.rs:2:18
  |
2 | fn foo((_a, _b): (i32, bool)) {}
  |                  ^^^^^^^^^^^